Monika After Story/game/definitions.rpy

```diff
@@ -2959,6 +2959,73 @@ init -1 python:
     def mas_isMonikaBirthday():
         return datetime.date.today() == mas_monika_birthday
 
+    def mas_isSpring(_date=datetime.date.today()):
+        """
+        Checks if given date is during spring
+        iff none passed in, then we assume today
+
+        Note: If persistent._mas_pm_live_north_hemisphere is none, we assume northern hemi
+
+        RETURNS:
+            boolean showing whether or not it's spring right now
+        """
+        _date = _date.replace(datetime.date.today().year)
+
+        if persistent._mas_pm_live_south_hemisphere:
+            return mas_fall_equinox <= _date < mas_winter_solstice
+        else:
+            return mas_spring_equinox <= _date < mas_summer_solstice
+
+    def mas_isSummer(_date=datetime.date.today()):
+        """
+        Checks if given date is during summer
+        iff none passed in, then we assume today
+
+        Note: If persistent._mas_pm_live_north_hemisphere is none, we assume northern hemi
+
+        RETURNS:
+            boolean showing whether or not it's summer right now
+        """
+        _date = _date.replace(datetime.date.today().year)
+
+        if persistent._mas_pm_live_south_hemisphere:
+            return mas_winter_solstice <= _date < mas_spring_equinox
```

Comment:
this was fine as it was before since you need to account for the season rolling over into a new year.
```suggestion
            return mas_winter_solstice <= _date or _date < mas_spring_equinox
```